getting-started/setup-building.rst

```diff
@@ -314,7 +314,7 @@ You can run the build of Python you've compiled with:
 See the `PCBuild readme`_ for more details on what other software is necessary
 and how to build.
 
-.. _Victor Stinner's guide: https://cpython-core-tutorial.readthedocs.io/en/latest/build_cpython_windows.html
+.. _Victor Stinner's guide: https://web.archive.org/web/20220907075854/https://cpython-core-tutorial.readthedocs.io/en/latest/build_cpython_windows.html
```

Comment:
Using https://github.com/vstinner/cpython_core_tutorial/blob/main/build_cpython_windows.rst is also an option.
Eventually I think it would be better to improve the Windows building instruction in the devguide and remove the link to Victor's tutorial.
In the meanwhile, using the archive.org link is ok.

Reply:
Yes correct, i asked this and i was advised to go with the archived one.

Reply:
> Eventually I think it would be better to improve the Windows building instruction in the devguide and remove the link to Victor's tutorial.
> In the meanwhile, using the archive.org link is ok.

Yeah, agreed, and seems Steve Dower had similar thoughts in his comments on my previous PR that I linked in the issue